CHARTER.md

```diff
@@ -11,6 +11,61 @@ The primary objectives of the resource aggregation working group includes themes
 
 ### 1.1 Process for Aggregating Assets
 
+This section describes the general approach for collecting assets for inclusion. We've identified types of resources that are described in more detail in later sections.
+
+#### 1.1.1 Terminology
+
+* **Resource**: an individual record that refers to a resource; assets are synonymous for resources
+* **Collection**: the set of assets that fall within a particular category; library, directory, and catalog are synonyms for a collection
+* **Aggregation**: the process of acquiring and storing assets into collections
+* **Contributor**: anybody from the community at large who provided an asset
+
+#### 1.1.2 Resource Collections
+
+A resource collection is the set of all assets that fall within a discrete category.
+
+Criteria for where resources are shared:
+- open access to the data
+- version control to account for change
+- scalable to accomodate a large number of assets
+
+The working group will make a recommendation for where to store resources.
+
+#### 1.1.3 Contribution
+
+Any member of the community should be able to submit a resource for inclusion so long as it adheres to the criteria for inclusion.
+
+Criteria for contribution:
+- anybody may submit a contribution to collections
+- tracking of contributions gives appropriate attribution
+- contributors agree to release of any licensing terms for origin of data
+
+The working group is responsible for insuring a framework for contribution is in place.
+
+#### 1.1.4 Review
+
+The value to the community is the ability to use a collection to meet their unique use cases. This value can be compromised if data quality is poor from sparse details or an excessive amount of records that do not support one of the intended goals.
+
+To execute a review:
+- review should occur within a reasonable amount of time
+- adherence to standards for data quality can be clearly judged
+- public access allows for community review and correction
+- protection is in place to promote fairness so that contribution is not made to spam the collections for individual gain
+- in cases of dispute, the owner of the resource is given priority
+- periodically an assessment should be made to ensure the review criteria best serves the devrel commmunity
+
```

Comment:
Do we want to define who can perform reviews, is it only managers, or can community contributors perform reviews? Do managers need to approve at that point?

Reply:
This may be a question of whether or not the charter should define that level of detail or not. I wavered a bit myself on whether to leave the charter as "we will define a review process that will look like..." versus the charter defining "this is the review process and it works like this...". 

I took the former interpretation, we need to establish a review process but _who_ specifically reviews is not yet defined. It could be a role we define in the structure of the working group which may not be mutually exclusive to managers.

Reply:
This basically:
```
The working group will define criteria and a recommendation for how the review process works.
```